cylc/flow/cfgspec/globalcfg.py

```diff
@@ -971,59 +972,84 @@
                 ssh
                    Use non-interactive ssh for task communications
             ''')
-            # TODO ensure that it is possible to over-ride the following three
-            # settings in workflow config.
-            Conf('submission polling intervals', VDR.V_INTERVAL_LIST, desc=f'''
+            Conf('submission polling intervals',
+                 VDR.V_INTERVAL_LIST,
+                 [DurationFloat(900)],
+                 desc='''
                 List of intervals at which to poll status of job submission.
 
-                {MOVEDFROMJOB}
-
                 Cylc can poll submitted jobs to catch problems that
                 prevent the submitted job from executing at all, such as
-                deletion from an external job runner queue. Routine
-                polling is done only for the polling ``task communication
-                method`` unless workflow-specific polling is configured in
-                the workflow configuration. A list of interval values can be
-                specified as for execution polling but a single value
-                is probably sufficient for job submission polling.
+                deletion from an external job runner queue. A list of interval
+                values can be specified, with the last value used repeatedly
+                until the task starts running.
+                Multipliers can be used as shorthand as in the example below.
 
                 Example::
 
-                   5*PT1M, 10*PT5M
+                   5*PT2M, PT5M
+
+                Note that if the polling
+                :cylc:conf:`global.cylc[platforms][<platform name>]
+                communication method`
+                is used then Cylc relies on the polling to detect when a task
+                starts running so you are likely to want to configure more
+                frequent polling.
             ''')
-            Conf('submission retry delays', VDR.V_INTERVAL_LIST, None, desc=f'''
-            {MOVEDFROMJOB}
+            Conf('submission retry delays', VDR.V_INTERVAL_LIST, None,
+                 desc='''
+                Cylc can automate resubmission of a task job which has not
+                been submitted sucessfully.
+
+                Execution retry delays are a list of ISO 8601
+                durations/intervals which tell Cylc how long to wait before
+                resubmitting a failed job.
+
+                Each time Cylc resubmits a task job it will increment the
+                variable ``$CYLC_TASK_TRY_NUMBER`` in the task execution
+                environment. ``$CYLC_TASK_TRY_NUMBER`` allows you to vary task
+                behavior between submission attempts.
```

Comment:
```suggestion
                Each time Cylc resubmits a job it increments the
                variable ``$CYLC_TASK_TRY_NUMBER`` in the job execution
                environment. This allows you to vary task
                behavior between submission attempts.
```

Reply:
```suggestion
                The scheduler makes ``$CYLC_TASK_TRY_NUMBER``
                available in the job execution environment. This allows you
                to vary task behavior between submission attempts.
```

Reply:
Users need to know the meaning of try number (and submit number) and when exactly it increments. So I wouldn't say that's only the "inner workings" of Cylc.

Reply:
OK. Fair enough.

Reply:
CYLC_TASK_TRY_NUMBER refers to execution - Remove!

Reply:
```suggestion
```

Reply:
Ohh, good point!!